client/components/mma/shared/SupporterPlusBenefits.tsx

```diff
@@ -0,0 +1,69 @@
+import { css } from '@emotion/react';
+import { from, palette, space, textSans } from '@guardian/source-foundations';
+import { SvgTickRound } from '@guardian/source-react-components';
+
+export const SupporterPlusBenefitsSection = (props: {
+	hidden?: boolean;
```

Comment:
I'm not sure if this really needs a `hidden` prop as it's only used in a single place. You could wrap it in a `div` in the one place where it's used and hide that.